SCClassLibrary/Common/Files/SoundFile.sc

```diff
@@ -407,10 +407,10 @@ SoundFile {
 
 	cue { | ev, playNow = false, closeWhenDone = false |
 		var server, packet, defname = "diskIn" ++ numChannels, condition, onClose;
-		ev = ev ? ();
+		ev = ev !? `_ ? `();
```

Comment:
I don't think that you really need a `Ref`. The event is already an object you can refer to.